src/App.js

```diff
@@ -1,19 +1,23 @@
-import React, { Component } from 'react';
-import logo from './logo.svg';
-import './App.css';
+//DEPENDENCIES
+  import React, { Component } from 'react';
+  import logo from './logo.svg';
+  import './App.css';
+
+//CONTAINERS
+  import InternalContactOverview from './containers/internalContactOverview'
+
+//MATERIALUI
+  //PROVIDER
+    import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider';
 
 class App extends Component {
   render() {
     return (
-      <div className="App">
-        <header className="App-header">
-          <img src={logo} className="App-logo" alt="logo" />
-          <h1 className="App-title">Welcome to React</h1>
-        </header>
-        <p className="App-intro">
-          To get started, edit <code>src/App.js</code> and save to reload.
-        </p>
-      </div>
+      <MuiThemeProvider>
```

Comment:
Wow, I did't know that we need to add this :grimacing:

Reply:
me neiiiither